helpers/testing/TestCases.py

```diff
@@ -0,0 +1,68 @@
+from django.test.client import RequestFactory
+from django.test import TestCase
+from django.contrib.auth.models import User
+
+
+"""
+Used when tests require specify which user is logged in
+Default is: super user
+"""
+
+
+class ViewTestCase(TestCase):
+    SCHOOL_ADMIN_CREDENTIALS = {
+        "username": "schooladmin@emis.com",
+        "password": "password",
+    }
+
+    SUPER_USER_CREDENTIALS = {
+        "username": "superuser@emis.com",
+        "password": "password",
+    }
+
+    @classmethod
+    def setUpClass(cls):
+        super(ViewTestCase, cls).setUpClass()
+
+    def setUp(self):
+        super(ViewTestCase, self).setUp()
+        self.login_super_user()
+
+    def ensure_accounts_exist(self, *args):
+        for item in args:
+            getattr(self, item)
+
+    @property
+    def school_admin_account(self):
+        if not hasattr(self, "_school_admin_account"):
+            self._school_admin_account, created = User.objects.get_or_create(
+                username=self.SCHOOL_ADMIN_CREDENTIALS["username"]
+            )
+            self._school_admin_account.set_password(
+                self.SCHOOL_ADMIN_CREDENTIALS["password"]
+            )
+            self._school_admin_account.save()
+        return self._school_admin_account
+
+    def login_school_admin_account(self):
+        self.ensure_accounts_exist("school_admin_account")
+        request = RequestFactory().get("/a/b/c")
+        return self.client.login(**self.SCHOOL_ADMIN_CREDENTIALS, request=request)
+
+    @property
+    def super_user_account(self):
```

Comment:
It looks like we've plugged a superuser into the DB with trivial credentials, which is fine for testing, but I don't know why we would want it to persist after the test.  Can we drop this user from the DB after testing with it is complete?

Same comment applies to the school admin.

Reply:
Nevermind, after reading more and testing this, I see that this occurs in a separate DB instance automatically.